source/includes/_lips_contacts.md

```diff
@@ -320,3 +321,170 @@ curl -L -X DELETE 'https://data-connect-lips.gannettdigital.com/contacts/63807'
+|`last_name`| string | Last Name of the contact |
+|`address`| string | Street address of the contact |
+|`city`| string | City of the contact |
+|`state`| string | State/province of the contact |
+|`postal`| string | Zip Code of the contact |
+|`email`| string | Email address of the contact |
+|`phone_numbers`|Array|phone number data. ('phone_type' phone_type of the contact, 'normalized_number' the normalized number of the phone number, 'number' the number of the phone number record, 'created_at' when the phone number was created, 'updated_at' when the phone number was updated )|
+|`master_capaign_id`| string | ID of the Campaign that generated the event |
+|`master_advertiser_id`| string | ID of the Advertiser the event is for |
+|`influencing_campaign_id`| string | ID of the Campaign that influenced the event |
+|`wpc_id`| string | ID of the Web Puiblisher Campaign that generated the event |
+|`influencing_wpc_id`| string | ID of the Web Puiblisher Campaign that influenced the event |
+|`referrer_type`| string | Capture referrer type for the event |
+|`page_url`| string | Page URL for the event |
+
+If `event_type` is FpdEvent, the following fields are also returned:
+
+| Field Name | Datatype | Description |
+|---|---|---|
+|`call_recording_url`| String | url used to match the contact|
+|`phone_number`| String | normalized phone number used to match the contact|
+
+If `event_type` is CallEvent, the following fields are also returned:
+
+| Field Name | Datatype | Description |
+|---|---|---|
+|`call_recording_url`| String | url used to match the contact|
+|`phone_number`| String | normalized phone number used to match the contact|
+|`call_duration` | String | duration of call in seconds |
+|`call_recording_url` | String | call recording url |
+|`caller_number` | String | Phone Number of the caller |
+|`dialed_number` | String | Phone Number the caller dialed |
+|`target_number` | String | Phone Number the call was redirected to|
+
+If `event_type` is ChatEvent, the following fields are also returned:
+
+| Field Name | Datatype | Description |
+|---|---|---|
+|`provider`| String | The chat provider |
+|`transcript`| String | Transcript of the chat|
+
+If `event_type` is FormEvent, the following fields are also returned:
+
+| Field Name | Datatype | Description |
+|---|---|---|
+|`referring_url` |String| URL of the referring page|
+|`cvt_type` |String| CVT Type|
+|`entry_type` |String| CVT Entry Type |
+|`page_cvt_id` |String| Page CVT ID |
+|`content` |String| Content of for fill (truncated to 80 chars) |
+|`extra_fields` |String| JSON of "extra fields" present on the form |
+
+#### Example Curl
+
+```
+curl -L -g -X GET 'https://data-connect-lips.ganettdigital.com/contacts/2626/events?page=1&per_page=2' \
+  -H 'Accept: */*' \
+  -H 'Authorization: token 1b01Secret' \
+  -H 'Content-Type: application/json' \
+  -H 'x-api-key: SJNPPSecret'
+```
+
+#### Example Response
+
+```javascript
+{
+    "events": [
```

Comment:
sadly, this is a bad example to be the only example for this API.  You are showing the events structure, but this is the Contacts API.